nntrainer/tensor/manager.h

```diff
@@ -441,11 +435,21 @@ class Manager {
   std::vector<std::unique_ptr<Var_Grad>>
     tensors_v2; /**< extra tensors required by the layers */
 
-  std::unordered_map<std::string, GraphNode::ExecutionOrder>
+  /** @todo: combine the list of the weights/var_grad to a common list */
```

Comment:
Wonder if this is done for the upcoming PRs

Reply:
This will be done after graph optimizations related to memory consumptions are done.